--- a/CMake/UseCopyright.cmake
+++ b/CMake/UseCopyright.cmake
@@ -18,8 +18,10 @@
   # need to raise an error if COPYRIGHT_MODULE_FILENAME is not set...
   if(EXISTS ${COPYRIGHT_MODULE_FILENAME} )
     foreach(filename ${ARGN})
-      file(READ ${filename} content)
-      file(APPEND ${COPYRIGHT_MODULE_FILENAME} ${content})
+	  if(EXISTS ${filename} )
+      	file(READ ${filename} content)
+        file(APPEND ${COPYRIGHT_MODULE_FILENAME} ${content})
+	  endif()
     endforeach()
   endif()
 endmacro()
