--- gdcm-2.0.18/Utilities/VTK/CMakeLists.txt.orig	2011-11-11 23:49:15.294924647 -0200
+++ gdcm-2.0.18/Utilities/VTK/CMakeLists.txt	2011-11-12 00:13:45.236789586 -0200
@@ -38,6 +38,8 @@ SET(vtkgdcm_LIBS
   vtkCommon
   vtkIO
   vtkImaging
+  vtkFiltering
+  vtkWidgets
 )
 IF(VTK_USE_RENDERING)
   SET(vtkgdcm_LIBS
@@ -429,7 +431,9 @@ IF(GDCM_WRAP_PYTHON)
     SET_TARGET_PROPERTIES(${VTKGDCM_NAME}Python PROPERTIES ${GDCM_LIBRARY_PROPERTIES})
     TARGET_LINK_LIBRARIES(${VTKGDCM_NAME}PythonD ${VTKGDCM_NAME})
     FOREACH(c ${vtkgdcm_LIBS})
-      TARGET_LINK_LIBRARIES(${VTKGDCM_NAME}PythonD ${c}Python${DEXTENSION})
+
+      TARGET_LINK_LIBRARIES(${VTKGDCM_NAME}PythonD ${c}Python${DEXTENSION} vtkCommon vtkIO vtkPythonCore python2.7)
+
     ENDFOREACH(c)
     TARGET_LINK_LIBRARIES(${VTKGDCM_NAME}Python ${VTKGDCM_NAME}PythonD)
     # Python extension modules on Windows must have the extension ".pyd"
--- gdcm-2.0.18/Utilities/VTK/Applications/CMakeLists.txt.orig	2011-11-12 00:43:20.959823899 -0200
+++ gdcm-2.0.18/Utilities/VTK/Applications/CMakeLists.txt	2011-11-12 00:45:40.862453690 -0200
@@ -33,7 +33,7 @@ FOREACH(app ${GDCM_VTK_APPS})
   TARGET_LINK_LIBRARIES(${app} ${VTKGDCM_NAME})
   TARGET_LINK_LIBRARIES(${app} gdcmDSED gdcmMSFF gdcmCommon)
   IF(VTK_USE_RENDERING)
-    TARGET_LINK_LIBRARIES(${app} vtkRendering)
+    TARGET_LINK_LIBRARIES(${app} vtkRendering vtkCommon vtkIO vtkImaging vtkFiltering)
 
   IF( "${VTK_MAJOR_VERSION}.${VTK_MINOR_VERSION}" GREATER 5.0 )
     TARGET_LINK_LIBRARIES(${app} vtkWidgets)
